sys/fuchsia/process.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright 2022 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# See: https://cs.opensource.google/fuchsia/fuchsia/+/main:zircon/vdso/process.fidl
+
+include <zircon/syscalls.h>
+
+resource zx_process[zx_handle]
+
+zx_process_exit(ret_code int64)
+
+zx_process_create(job zx_job, name ptr[in, string], name_len len[name], options const[0], proc_handle ptr[out, zx_process], vmar_handle ptr[out, zx_vmar])
```

Comment:
`options` also now has a valid value of `ZX_PROCESS_SHARED` (which is equal to 1). How about changing that arg's type to `flags[...]`?

(I think that option is only interesting if `zx_process_create_shared` is enabled, so that update could be a TODO for https://bugs.fuchsia.dev/p/fuchsia/issues/detail?id=118199 instead if you prefer.)

Reply:
Done.